tests/testthat/test_ncdf.R

```diff
@@ -221,3 +223,11 @@ test_that("4d not 4d", {
   expect_equal(dim$level$to, 2)
 
 })
+
+test_that("units are right with lcc km", {
+	f <- system.file("nc/lcc_km.nc", package = "stars")
```

Comment:
I converted all the actual data in this file to 0s so it's only ~35k and we can atleast do the spatial tests on it.

```
:History = "Wed May  4 21:41:21 2022: ncks -O -7 --cnk_plc=all --cnk_map=dmn -L 4 prcp_annttl_1980_ncss8.nc prcp_annttl_1980_ncss9.nc\n",
           "Wed May  4 21:41:05 2022: ncap2 -s prcp=prcp*0 prcp_annttl_1980_ncss2.nc prcp_annttl_1980_ncss8.nc\n",
           "Wed May  4 21:27:51 2022: ncks -x -v lon,lat prcp_annttl_1980_ncss.nc prcp_annttl_1980_ncss2.nc\n",

```